src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -792,6 +796,23 @@ function showCurrent(state)
         return $this->dumpHeader = preg_replace('/\s+/', ' ', $line).'</style>'.$this->dumpHeader;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function dumpString(Cursor $cursor, $str, $bin, $cut)
+    {
+        if ('' === $str && isset($cursor->attr['img-data'], $cursor->attr['content-type'])) {
+            // we can not do fancy CSS here since we are in a <pre>
+            $this->dumpKey($cursor);
+            $this->line .= "\n";
+            $this->line .= str_repeat(' ', $cursor->depth + 8);
```

Comment:
We shoud use `$this->dumpLine` instead of dealing with indentation manually.

Reply:
I tried, It messes up everything:
![image](https://user-images.githubusercontent.com/408368/61872683-4c74a480-aee4-11e9-91cf-c5088f5a05ff.png)


As you can see, I built a weird HTML, because the code is in a `<pre>`:
```
previousProperty: true
image:
           [real image]
           [that uses  ]
           [this place ]
```
Usually, thing are dumped like that `image: [value]` but with a picture it renders like that:
```
previousProperty: true
            [real image]
            [that uses  ]
image: [this place ]
```